svgjs.resize/svgjs.resize.d.ts

```diff
@@ -0,0 +1,15 @@
+// Type definitions for svgjs.resize
+// Project: http://www.svgjs.com/
+// Definitions by: Kevin Gutiérrez <https://github.com/jkevingutierrez>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Reference: https://github.com/Fuzzyma/svg.resize.js
+
+
+declare namespace svgjs {
+	export interface Element {
```

Comment:
i do not think does what you intended to do. this does not augment the global `Element` interface.
either declare it global, i.e.

```ts
interface Element { ... }
```
or use  instead.

```ts
declare global { 
    interface Element { ... }
}
```